### DISCUSSION TITLE
Is it possible to specify parts of a screenshot so that they are not compared in the image-plugin?

### DISCUSSION THREAD

**Discussion by Stefankoff** [appium/appium#19881] [Q&A]
I test applications in which some parts of the screens (photos, videos, comments) are dynamic, i.e. each time you enter the screen, you see different photo or text, etc.
Is it possible to hide such parts of the screen for reliable results using an image-plugin, or is such a feature planned?

**Comment by mykola-mokhnach** [comment]
it is possible to lookup elements inside other elements, like findElement(x).findElement(y), so the lookup area for the picture y would be limited by image x.
